csm_web/scheduler/email/email_content/signature.html

```diff
@@ -0,0 +1,3 @@
+<p>
+  Want to unsubscribe? Click <a href="http://localhost:8000/#/profilePage">here</a> to visit your profile settings.
```

Comment:
This link should be made to be not localhost; ideally, this should either be in a constant field and injected, or can be hard-coded here, with some documentation regarding it.